### PR TITLE
fix optional install instructions in the Summit profile

### PR DESCRIPTION
**Pull request by sbastrakov**
#3007 has some paths of @ax3l , I assume a leftover.
Thanks @arghyac007 for pointing out.

### CODE REVIEW

File: etc/picongpu/summit-ornl/gpu_picongpu.profile.example [hunk 1]
```diff
@@ -33,20 +33,20 @@ module load boost/1.66.0
 module load hdf5/1.10.3
 module load adios/1.13.1-py2 c-blosc zfp sz lz4
 
-# download libSplash and compile it yourself from
+# optionally download libSplash and compile it yourself from
 #   https://github.com/ComputationalRadiationPhysics/libSplash/
-export Splash_DIR=$PROJWORK/$proj/huebl/sw/splash-1.7.0
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$Splash_DIR/lib
+# export Splash_DIR=<your libSplash install directory>
```
Comment (ax3l) [inline etc/picongpu/summit-ornl/gpu_picongpu.profile.example]: These days, e.g. in the `cori-nersc` profile, I usually recommend some path such as
```bash
export Splash_ROOT=${HOME}/sw/libSplash-1.7.0
```

Another example can be seen in `pizdaint-cscs` which is close to that.

Just trying to keep it similar so we are not introducing yet another naming scheme.

File: etc/picongpu/summit-ornl/gpu_picongpu.profile.example [hunk 2]
```diff
@@ -33,20 +33,20 @@ module load boost/1.66.0
...
 
-# install pngwriter yourself:
-#   https://github.com/pngwriter/pngwriter#install
 module load zlib/1.2.11
 module load libpng/1.6.34 freetype/2.9.1
-export PNGwriter_DIR=$PROJWORK/$proj/huebl/sw/pngwriter-0.7.0
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$PNGwriter_DIR/lib
+# optionally install pngwriter yourself:
+#   https://github.com/pngwriter/pngwriter#install
+# export PNGwriter_DIR=<your pngwriter install directory>
```
Comment (ax3l) [inline etc/picongpu/summit-ornl/gpu_picongpu.profile.example]: Just an idea: you can ask user support about a PNGwriter module to ease this.
I added PNGwriter to libSplash (as `pngwriter`) and OLCF uses Spack for their module installs.

Btw, `libsplash` is another one :)

File: etc/picongpu/summit-ornl/gpu_picongpu.profile.example (hunk 1 above)
Comment (sbastrakov) [reply etc/picongpu/summit-ornl/gpu_picongpu.profile.example]: Thanks, I will change to `_ROOT`. Before just kept the naming scheme in that file.

**Comment by sbastrakov** [comment]
@ax3l thanks, I changed `_DIR` to `_ROOT` and added suggested install paths. Rebased.

File: etc/picongpu/summit-ornl/gpu_picongpu.profile.example [hunk 3]
```diff
@@ -33,20 +33,20 @@ module load boost/1.66.0
 module load hdf5/1.10.3
 module load adios/1.13.1-py2 c-blosc zfp sz lz4
 
-# download libSplash and compile it yourself from
+# optionally download libSplash and compile it yourself from
 #   https://github.com/ComputationalRadiationPhysics/libSplash/
-export Splash_DIR=$PROJWORK/$proj/huebl/sw/splash-1.7.0
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$Splash_DIR/lib
+# export Splash_ROOT=<your libSplash install directory, e.g., ${HOME}/sw/libSplash-1.7.0>
```
Comment (ax3l) [inline etc/picongpu/summit-ornl/gpu_picongpu.profile.example]:
```suggestion
# export Splash_ROOT=<your libSplash install directory> e.g., ${HOME}/sw/libSplash-1.7.0>
```
But I am not sure the in-line ` e.g. more text` will help users that just comment-in the line...
maybe
```suggestion
# export Splash_ROOT=<your libSplash install directory>  # e.g., ${HOME}/sw/libSplash-1.7.0>
```
instead

File: etc/picongpu/summit-ornl/gpu_picongpu.profile.example [hunk 4]
```diff
@@ -33,20 +33,20 @@ module load boost/1.66.0
...
 
-# install pngwriter yourself:
-#   https://github.com/pngwriter/pngwriter#install
 module load zlib/1.2.11
 module load libpng/1.6.34 freetype/2.9.1
-export PNGwriter_DIR=$PROJWORK/$proj/huebl/sw/pngwriter-0.7.0
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$PNGwriter_DIR/lib
+# optionally install pngwriter yourself:
+#   https://github.com/pngwriter/pngwriter#install
+# export PNGwriter_ROOT=<your pngwriter install directory, e.g., ${HOME}/sw/pngwriter>
```
Comment (ax3l) [inline etc/picongpu/summit-ornl/gpu_picongpu.profile.example]:
```suggestion
# export PNGwriter_ROOT=<your pngwriter install directory>  # e.g., ${HOME}/sw/pngwriter>
```

File: etc/picongpu/summit-ornl/gpu_picongpu.profile.example (hunk 3 above)
Comment (sbastrakov) [reply etc/picongpu/summit-ornl/gpu_picongpu.profile.example]: Okay, I will follow the latter option.

**Pull request merged**